packages/@react-spectrum/actiongroup/docs/ActionGroup.mdx

```diff
@@ -34,12 +34,12 @@ category: Buttons
 
 <HeaderInfo
   packageData={packageData}
-  componentNames={['ActionGroup']} />
+  componentNames={['ActionGroup', 'Item']} />
 
 ## Example
 
 ```tsx example
-<ActionGroup>
+<ActionGroup onAction={(key) => console.log(key)}>
```

Comment:
Console.log is hard to see when you don't have devtools open.

```suggestion
<ActionGroup onAction={(key) => alert(key)}>
```

Reply:
yes but I find alert to be very annoying and interrupting. I would like 1 or 2 examples as alerts, to be very obvious, then the rest as logs so that they don't get in the way of playing with the docs.